apps/daimo-mobile/src/i18n/languages/en.ts

```diff
@@ -444,10 +446,13 @@ export const en = {
   // DepositScreen.tsx
   deposit: {
     screenHeader: () => `Deposit or Withdraw`,
+    go: () => `Go`,
```

Comment:
@Ethanol48 caught these newly added Landline text -- could you add to Spanish? Thank you so much!